### ISSUE TITLE
[Feature]: Render Release Notes in Markdown in the same way as READMEs are

### ISSUE THREAD

**Issue by baronfel** [NuGet/NuGetGallery#8889]
### Related Problem

As a package user, I enjoy the new README markdown support in NuGet.org, but when I go to packages I already use to see what's changed between releases, the Release Notes for that package show unformatted markdown, which is difficult to read compared to the rendered form.

### The Elevator Pitch

The new Release Notes and README support on the NuGet website is great! READMEs in particular are a great experience for people using the gallery for package discovery.  The Release Notes tab looks a little drab by comparison, unfortunately. It would be great if the Release Notes tab could render Markdown in the same way as READMEs in order to present a unified experience to a viewer. Many packages use markdown for links, bulleted lists, and sometimes even embedded images to demonstrate changes, and unifying that would be a marked (heh) improvement in my view.

### Additional Context and Details

* NPM - no support for release notes/changelog, just a static list of versions
* Crates.io - no support for release notes/changelog, provides a list of versions with release metadata and clickable links
* Maven Central - no support for release notes/changelog, a list of versions is provided to swap views of metadata

**Comment by gregsdennis** [comment]
Yes, this.  Absolutely this.  I'm amazed release notes as markdown weren't done to begin with given readme already supports it.

**Comment by AndrewTriesToCode** [comment]
Would love to see **span** tag support. I use them as markers for dynamic updates of content in my build process and they look terrible in NuGet readme view:
https://www.nuget.org/packages/Finbuckle.MultiTenant#readme-body-tab
